### ISSUE TITLE
502 from Google not handled gracefully

### ISSUE THREAD

**Issue by jonahwh** [qpowell/google_places#106]
v1.0.0

While making a request to the Google Places API, we received a response from Google with a status of 502, with the following body:

```
<html lang=en> 
 <meta charset=utf-8> 
 <meta name=viewport content="initial-scale=1, minimum-scale=1, width=device-width"> 
 <title>Error 502 (Server Error)!!1</title> 
 <style> 
   *snip*
 </style> 
 <a href=//www.google.com/><span id=logo aria-label=Google></span></a> 
 <p><b>502.</b> <ins>That’s an error.</ins> 
 <p>The server encountered a temporary error and could not complete your request.<p>Please try again in 30 seconds.  <ins>That’s all we know.</ins>
...
```

This error is unhandled by the library, and a `JSON::ParserError` is thrown by
https://github.com/qpowell/google_places/blob/cb33d8bae7c8ae850d8cbb47a0b350eaecf5e9eb/lib/google_places/request.rb#L351

**Comment by edwinwills** [comment]
@jonahwh thanks for letting us know about this error - I've stuck up a PR at https://github.com/qpowell/google_places/pull/107 that should solve this in the future. @qpowell mind giving it a quick once-over?

**Comment by qpowell** [comment]
@jonahwh Thanks for reporting this. Closing this ticket as it was addressed in #107
